mod/groups/views/default/group/format/invitationrequest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Group view for an invitation request
+ *
+ * @uses $vars['entity'] Group entity
+ */
+
+$user = elgg_get_logged_in_user_entity();
+$group = elgg_extract('entity', $vars);
+
+if (!$group instanceof \ElggGroup) {
+	return true;
+}
+
+$icon = elgg_view_entity_icon($group, 'small');
+
+$accept_url = elgg_http_add_url_query_elements('action/groups/join', array(
+	'user_guid' => $user->guid,
+	'group_guid'=> $group->guid,
+	));
+
+$accept_button = elgg_view('output/url', array(
+	'href' => $accept_url,
+	'is_action' => true,
+	'text' => elgg_echo('accept'),
+	'class' => 'elgg-button elgg-button-submit',
+	'is_trusted' => true,
+		));
+
+$delete_url = elgg_http_add_url_query_elements('action/groups/killinvitation', array(
+	'user_guid' => $user->guid,
+	'group_guid' => $group->guid,
+));
+
+$delete_button = elgg_view('output/url', array(
+	'href' => $delete_url,
+	'is_action' => true,
+	'confirm' => elgg_echo('groups:invite:remove:check'),
+	'text' => elgg_echo('delete'),
+	'class' => 'elgg-button elgg-button-delete mlm',
+		));
+
+$buttonbank = $accept_button . $delete_button;
```

Comment:
I am done for today. Going for some deserved clubbing. Will finish Monday :)
On Apr 11, 2015 7:35 PM, "Jeroen Dalsem" <notifications@github.com> wrote:

> In mod/groups/views/default/group/format/invitationrequest.php
> <https://github.com/Elgg/Elgg/pull/8154#discussion_r28196865>:
>
> > +		));
> > +
> > +$delete_url = elgg_http_add_url_query_elements('action/groups/killinvitation', array(
> > +	'user_guid' => $user->guid,
> > +	'group_guid' => $group->guid,
> > +));
> > +
> > +$delete_button = elgg_view('output/url', array(
> > +	'href' => $delete_url,
> > +	'is_action' => true,
> > +	'confirm' => elgg_echo('groups:invite:remove:check'),
> > +	'text' => elgg_echo('delete'),
> > +	'class' => 'elgg-button elgg-button-delete mlm',
> > +		));
> > +
> > +$buttonbank = $accept_button . $delete_button;
>
> or do it the same... they will be overriden...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8154/files#r28196865>.
>